api/src/main/java/org/pmiops/workbench/utils/WorkspaceMapper.java

```diff
@@ -1,36 +1,40 @@
 package org.pmiops.workbench.utils;
 
-import com.google.common.collect.ImmutableSet;
-import java.util.List;
-import java.util.Set;
+import static org.mapstruct.NullValuePropertyMappingStrategy.*;
+
+import org.mapstruct.CollectionMappingStrategy;
 import org.mapstruct.Mapper;
 import org.mapstruct.Mapping;
-import org.pmiops.workbench.db.model.DbStorageEnums;
+import org.mapstruct.MappingTarget;
 import org.pmiops.workbench.db.model.DbWorkspace;
 import org.pmiops.workbench.firecloud.model.FirecloudWorkspace;
 import org.pmiops.workbench.model.CdrVersion;
 import org.pmiops.workbench.model.ResearchPurpose;
-import org.pmiops.workbench.model.SpecificPopulationEnum;
 import org.pmiops.workbench.model.Workspace;
 import org.pmiops.workbench.utils.mappers.CommonMappers;
 
 @Mapper(
     componentModel = "spring",
+    collectionMappingStrategy = CollectionMappingStrategy.TARGET_IMMUTABLE,
```

Comment:
This is only used when there's a mapping target and a new object is not created. In this case, MapStruct will default to using `get() -> clear() ->  addAll()`. This doesn't work if the object's getter returns a copy of the collection instead of a reference to the real data, which is what we do for our enum sets in DbWorkspace.